server/consts_t.go

```diff
@@ -20,4 +20,7 @@ const (
 
 	ReportMigrationFrequency        = 100 * time.Millisecond
 	WattTimeWeekDataUpdateFrequency = 1000 * time.Millisecond
+
+	apiArchiveLimit = 1
+	apiArchiveRate  = 100 * time.Millisecond
```

Comment:
similar thing here probably. 100ms also seems really long for a testing based ratelimit

Reply:
I'll stick with limit 3 and pick a rate that makes sense